types/three/three-core.d.ts

```diff
@@ -2033,6 +2045,29 @@ export class SpotLight extends Light {
     penumbra: number;
 }
 
+/**
+ * This light gets emitted uniformly across the face a rectangular plane.
+ * This can be used to simulate things like bright windows or strip lighting.
+ */
+export class RectAreaLight extends Light {
+    /**
+     * @param color Hexadecimal color of the light. Default is 0xffffff (white)
+     * @param intensity Numeric value of the light's strength/intensity. Default is 1
+     * @param width Width of the light. Default is 10
+     * @param height Height of the light. Default is 10
+     */
+    constructor(color?: number|string, intensity?: number, width?: number, height?: number);
```

Comment:
looks like color can also be a `THREE.Color`,
It ends up here:
https://github.com/mrdoob/three.js/blob/74d0bdc64271f3e7f1b23c331a733433b1ea9915/src/math/Color.js#L37